crates/boundless-market/src/order_stream_client.rs

```diff
@@ -86,6 +88,9 @@ pub struct Order {
     /// Order request
     #[schema(value_type = Object)]
     pub request: ProofRequest,
+    /// Request digest
+    #[schema(value_type = Object)]
+    pub request_digest: B256,
```

Comment:
This is a breaking change, but not on a contract level, but if we update, then we would need to update the order stream service and require existing brokers update

Reply:
#362 is this change in the backports

Reply:
That commit has been dropped.